lib/widgets/post/widgets/post-body/widgets/post_body_media/post_body_media.dart

```diff
@@ -206,7 +206,7 @@ class OBPostBodyMediaState extends State<OBPostBodyMedia> {
     _setRetrievePostMediaInProgress(true);
     try {
       _retrievePostMediaOperation = CancelableOperation.fromFuture(
-          _userService.getMediaForPost(post: widget.post));
+          _userService.getMediaForPost(post: widget.post), onCancel: _onRetrievePostMediaOperationCancelled);
```

Comment:
_setRetrievePostMediaInProgress 

should also be called on success. so please call at the end of the try block

Reply:
there's a finally block which does that 🤔  ?